pkg/windows/windows.go

```diff
@@ -560,6 +563,9 @@ func (vm *windows) runBootstrapper() error {
 	if vm.instance.SetNodeIP {
 		wmcbInitializeCmd += " --node-ip=" + vm.GetIPv4Address()
 	}
+	if vm.platformType != "" {
+		wmcbInitializeCmd += " --platform-type=" + vm.platformType
+	}
```

Comment:
In what case will `platformType == ""`?

Reply:
Yeah, previously it could be empty, but now it always has some value. This check is not necessary anymore.